definitions/divorce/json/FixedLists.json

```diff
@@ -239,6 +239,12 @@
     "ListElementCode": "baliffService",
     "ListElement": "Bailiff Service"
   },
+  {
+    "LiveFrom": "01/01/2017",
+    "ID": "d8documentsReceivedEnum",
+    "ListElementCode": "coe",
+    "ListElement": "Certificate Of Entitlement"
+  },
```

Comment:
Where is this used?

Reply:
We are generating the Certificate of Entitlement now and it needs to be stored in CCD. There should be a Document Type identifier against it so it can be tracked for uniqueness.